src/main/java/org/jpeek/Report.java

```diff
@@ -95,8 +103,29 @@
      * @param name Name of the metric
      */
     Report(final XML xml, final String name) {
-        // @checkstyle MagicNumberCheck (1 line)
-        this(xml, name, new HashMap<>(0), 0.5d, 0.1d);
+        this(
+            xml,
+            name,
+            new HashMap<>(0),
+            Report.DEFAULT_MEAN,
+            Report.DEFAULT_SIGMA
+        );
+    }
+
+    /**
+     * Ctor.
+     * @param xml Skeleton
+     * @param name Name of metric
+     * @param args Params for XSL
+     */
+    Report(final XML xml, final String name, final Map<String, Object> args) {
+        this(
+            xml,
```

Comment:
@llorllale I think all these params fit on one line, no? If not, make them 2 lines, 1 param/line seems noisy.